### PR TITLE
New changes for chicago

### PR DESCRIPTION
**Pull request by psaweh**
Add new data to the file

### CODE REVIEW

**Comment by psaweh** [comment]
@pjbrown9

**Comment by psaweh** [comment]
@pjbrown9  could you please look at this

File: paulsaweh-chicago.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Chicago, IL
 ### Restaurants
---Taco Bell
+-Taco Bell
```
Comment (pjbrown9) [inline paulsaweh-chicago.md:3]: Please put these in alpha order

**Comment by psaweh** [comment]
@pjbrown9 please review

**Review by pjbrown9 — CHANGES_REQUESTED** [review]
The data should be in alpha order, it's easier on the eyes for the User.

**Review by psaweh — COMMENTED** [review]
I made the changes

**Review by pjbrown9 — APPROVED** [review]
looks great!

**Pull request merged**